modules/auxiliary/gather/cisco/cisco_rv320_config.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
```

Comment:
If you're going to include the scanner mixin, please move this to `aux/scanner`.

Reply:
I'm happy to take `Msf::Auxiliary::Scanner` out.  I don't think it does anything for me in this context.